dask/dataframe/io/tests/test_parquet.py

```diff
@@ -24,10 +24,8 @@
 
 try:
     import pyarrow as pa
-
-    check_pa_divs = pa.__version__ >= LooseVersion("0.9.0")
 except ImportError:
-    check_pa_divs = False
+    pa = False
```

Comment:
To prevent potential future errors, should ensure both symbols are defined in both cases.

```suggestion
    check_pa_divs = pa = False
```